plugins/woocommerce/changelog/feature-add-loading-state-to-schedule-button

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Add loading state functionality to Schedule button in the calendar
```

Comment:
```suggestion
Product Editor: Add loading state functionality to Schedule Product modal
```

Reply:
This changelog file actually needs to be for the `@woocommerce/product-editor` package instead of `plugins/woocommerce`.

Please remove this changelog and then create a new changelog file with the following command (from the root of the monorepo):

```
pnpm -r --filter=@woocommerce/product-editor run changelog add
```

I've also made a suggestion to the changelog message, to make it more clear what section of the UI is is referring to!

Reply:
@mattsherman I appreciate your review, the changelog file has been updated now.